types/operator_metadata.go

```diff
@@ -188,3 +186,23 @@ func isImageURL(urlString string) error {
 
 	return ErrInvalidImageExtension
 }
+
+func validateText(text string) error {
+	if len(text) == 0 {
+		return ErrEmptyText
+	}
+
+	if len(text) > 200 {
+		return ErrTextTooLong
+	}
+
+	// Regular expression to validate text
+	textPattern := regexp.MustCompile(`^[a-zA-Z0-9@#$%^&*()'_+\- =,.;:/?|~\[\]{}]*$`)
```

Comment:
This looks good, `regexp.MustCompile(`^[a-zA-Z0-9 .,;:?!'"\-_/()\[\]]+$`)` suggesting a more restricted version, please consider if this looks good.

Reply:
makes sense. let's start with strict and we can loosen it if needed